api/src/pcapi/routes/public/individual_offers/v1/endpoints.py

```diff
@@ -29,7 +30,9 @@ def _retrieve_venue_from_location(
             offerers_models.Venue.managingOffererId == offerer_id,
         ).one_or_none()
         if not venue:
-            raise api_errors.ApiErrors({"venueId": ["There is no Venue associated to your ApiKey"]}, status_code=404)
+            raise api_errors.ApiErrors(
+                {"venueId": ["there is no venue with this id associated to your api key"]}, status_code=404
```

Comment:
(détail) Si on peut chipoter sur la formulation, je m'y mets moi aussi :)

```suggestion
                {"venueId": ["There is no venue with this id associated to your API key"]}, status_code=404
```

API en majuscule parce que c'est un acronyme, et "T" majuscule en début de message car j'ai l'impression que c'est ce qu'on fait ailleurs.